packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -3338,6 +3338,115 @@ enum ClipboardStatus {
   notPasteable,
 }
 
+/// A [ValueNotifier] whose [value] indicates whether the current device supports the Live Text
+/// (OCR) function.
+///
+/// See also:
+///  * [LiveText], where the availability of Live Text input can be obtained.
+///  * [LiveTextInputStatus], an enumeration that indicates whether the current device is available
+///                           for Live Text input.
```

Comment:
Nit: I think usually the indentation looks like this, but if you've seen it this way elsewhere then don't worry about it.

```suggestion
///  * [LiveTextInputStatus], an enumeration that indicates whether the current device is available
///    for Live Text input.
```